spec/models/post_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject do
+    Post.create do |post|
+      post.title = 'Post 1'
+      post.text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+      post.author = User.create do |user|
+        user.name = 'John Doe'
+        user.photo = 'https://picsum.photos/200'
+        user.bio = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+      end
+    end
+  end
+
```

Comment:
- [ ] Great job so far 👏🏻 However, all custom methods in the models must have a corresponding unit test. Actually, you are missing a test to test the `update_posts_counter` method of your Post model, please add this ✍🏻

```ruby
  def update_posts_counter
    author.increment!(:posts_counter)
  end
```